mk/lint.mk

```
@@ -0,0 +1,15 @@
+.PHONY: install-pre-commit, install-golangci-lint, install-linter, lint, lint-all
+
+install-pre-commit:
+	pip install pre-commit
```

```suggestion
	pip3 install pre-commit
```

the last macos system does not have pip binary (neither a symbolic link to pip3); fedora 36 has `pip3`; not sure above the next statement so I write as a question: is it right to say that modern linux distros has python3 and pip3 binaries? anyway, `pip3` worked on fedora 36 and macos systems.